client/src/RatingsAndReviews/Reviews/AddReview.jsx

```diff
@@ -47,34 +69,66 @@ class AddReview extends React.Component {
     });
   }
 
+  onFileUpload(event) {
+    // convert file to base64 data
+    let fileReader = new FileReader();
+    fileReader.readAsDataURL(event.target.files[0], 'base64');
+    // once the fileReader is loaded
+    fileReader.onload = () => {
+      // get the result as url
+      let url = fileReader.result.substring(0);
+      // make a request to imgbb
+      getUrl(url)
```

Comment:
we both already know this - but seems to be some error happening in the get function i wrote for the images.  For whatever reason we are able to see whatever we post locally, but the url for imgBB is invalid - prob need to figure this out between both of us - ill keep you updated if i find anything